deps/build.jl

```diff
@@ -43,3 +44,46 @@ end
 compile(TZDATA_DIR, COMPILED_DIR)
 
 info("Successfully processed TimeZone data")
+
+
+if OS_NAME == :Windows #TODO change to window before commit
```

Comment:
What's up with the comment? After talking with @iamed2  I think using `@windows_only begin` here is probably preferable.